accessibilityTest/Test.ts

```diff
@@ -88,6 +88,7 @@ describe('Testing ...', () => {
     initialHTMLSetup();
     Coveo.Logger.disable();
     Coveo.SearchEndpoint.configureSampleEndpointV2();
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 15000;
```

Comment:
We've increased the timeout several times in recent weeks. While this is an easy way out, the side-effect s our tests take longer and longer to run. Could there be a deeper issue that we should address?

Are there specific tests that always hit the timeout? or is it random? Are the kinds of failures the same (e.g. only DOM queries, only api calls)? or random?

Reply:
I'm investigating this right now. I ran every test with a 500ms latency and added a `console.info` to see how much time each test takes to run, and some reach above 11000ms. If you want the log, I can upload it.

Reply:
No need for the log, but if the timeouts are limited to the same few tests, then those would be good targets to dig into to understand what is happening.

Reply:
While some tests take more time than others to run, at a 500ms latency, they almost all take 3000-9000ms with one or two exceptions at 11000. It seems like it would be difficult to target some specific tests. If you still think I should target specific tests, I'll further look into it.